app/views/shared/_no_pii_banner.html.erb

```diff
@@ -1,3 +1,3 @@
-<div class='padding-y-1 bg-secondary-darker text-white fs-12p line-height-1 center'>
+<div class='h1 padding-y-4 bg-secondary-darker text-white fs-12p line-height-1 center'>
```

Comment:
I vote for even bigger!! cover half the screen!

another idea I'd like us to consider is adding something inline next to the SSN field and other PII fields because of banner blindness

Reply:
Yeah, planning on cooking something like that up. This is the late in the evening quick win version.